### PR TITLE
Create Task7.py

### PR DESCRIPTION
**Pull request by AlexeyKozinets**

### CODE REVIEW

File: Tasks/Kozinets/Task7/Task7.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+a = [1,2,[2,4,[[7,8],4],6],[1,2,[2,4,[[7,8],4],6]]] #<- 68
+
+def list_sum(l):
+	j = 0
+	for i in l:
+		j += list_sum(i) if isinstance(i,list) else i
+	return j
+
+
+def fib_1(n): #return list of integers
```
Comment (EugeneDenkevich) [inline Tasks/Kozinets/Task7/Task7.py:10]: Realy nice solution!)

File: Tasks/Kozinets/Task7/Task7.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+def fib_4(n): #the_dich
+	if n != 1:
+		return fib_4(n - 1) + [1] if n == 2 else fib_4(n-1) + [fib_4(n-1)[-1] + fib_4(n-1)[-2]]
+	else:
+		return [0]
+
+def fib_5(n): #the_dich_V2
+	return fib_5(n-1) + [1] if n == 2 else fib_5(n-1) + [fib_5(n-1)[-1] + fib_5(n-1)[-2]] if n != 1 else [0]
```
Comment (EugeneDenkevich) [inline Tasks/Kozinets/Task7/Task7.py:43]: Realy difficult to understand for me, but logic is like in previous solutions. =) Like!)

File: Tasks/Kozinets/Task7/Task7.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	if n != 1:
+		j = fib_3(n-1)
+		return j + [1] if n == 2 else j + [j[-1] + j[-2]]
+	else:
+		return [0]
+
+
+def fib_4(n): #heresy(heavy load on the processor (for example try n = 9 and n = 13)) but working
+	if n != 1:
+		return fib_4(n - 1) + [1] if n == 2 else fib_4(n-1) + [fib_4(n-1)[-1] + fib_4(n-1)[-2]]
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kozinets/Task7/Task7.py:38]: hell yeah, welcome to the cult))

File: Tasks/Kozinets/Task7/Task7.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+	else:
+		return ['0']
+
+
+def fib_3(n): #shortest way
+	if n != 1:
+		j = fib_3(n-1)
+		return j + [1] if n == 2 else j + [j[-1] + j[-2]]
+	else:
+		return [0]
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kozinets/Task7/Task7.py:33]: I like this one

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Very good, all those variants are great, the score is 100

**Pull request merged**